skylog/storage/cloudspanner/schema.sdl

```diff
@@ -1,3 +1,13 @@
+-- Entries stores sequenced entries of the log, which also correspond to leaves
+-- of the Merkle Tree.
+CREATE TABLE Entries (
```

Comment:
Hm, I'm wondering if interleaving this into `TreeNodes` is a good idea (mapping between `EntryIndex` and `NodeID` is trivial).

Reply:
The upside of doing that is having `NodeHash` close to `Data`.

Reply:
OTOH, `Entries` is tailored for horizontal reads, whereas `TreeNodes` is (mostly) for vertical inclusion-proof-like reads. I don't want to mix the two in one table, so keeping them separate for now.